src/Monitor/Monitor/help/Get-AzActivityLogAlert.md

```diff
@@ -128,8 +128,6 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 [Set-AzActivityLogAlert](./Set-AzActivityLogAlert.md)
 
-[Update-AzActivityLogAlert](./Update-AzActivityLogAlert.md)
```

Comment:
Is this removed by mistake?

Reply:
The removal was intentional. The link is invalid due to no PowerShell command by that name existing in any of the Az PowerShell modules.